workflows/references/main.nf

```diff
@@ -99,6 +103,17 @@ workflow REFERENCES {
         versions = versions.mix(MSISENSORPRO_SCAN.out.versions.first())
     }
 
+    if (tools && tools.split(',').contains('hisat2')) {
+        HISAT2_EXTRACTSPLICESITES(input.gtf)
+        splicesites = input.splice_sites ?: HISAT2_EXTRACTSPLICESITES.out.txt
+
+        HISAT2_BUILD(input.fasta, input.gtf, splicesites)
+
+        hisat2 = HISAT2_BUILD.out.index
```

Comment:
do you wanna emit the splicesites out too ?

Reply:
or not that imp for downstream ?

Reply:
yeah, I'll do later, it's planned